bonobo/nodes/basics.py

```diff
@@ -314,6 +316,32 @@ def _Format(context, bag):
     return _Format
 
 
+@transformation_factory
+def MapFields(callFunc):
+    # TODO: Figure out how to more intelligently handle correctable errors
+    # Use this wrapper to call the function that we pass in and handle the error
+    # For now, handle the error by returning the unmodified value
+    def _wrapfunc(func, val):
+        try:
+            retval = func(val)  # Attempt to use the function
+        except:
```

Comment:
We should never use a "catchall" except, instead, catch `Exception`.

Not sure this behaviour is wanted though, as it may come to unpredictable behaviour.